testing/regress/ecl/split_test.ecl

```diff
@@ -0,0 +1,574 @@
+/*##############################################################################
+
+    Copyright (C) 2017 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+//nohthor
+//noroxie
+
+//class=spray
+//class=copy
+//class=dfuplus
+
+import Std.File AS FileServices;
+import $.setup;
+prefix := setup.Files(false, false).QueryFilePrefix;
+
+dropzonePath := '/var/lib/HPCCSystems/mydropzone/' : STORED('dropzonePath');
+
+unsigned VERBOSE := 0;
+unsigned KEEP_FILES := 0;
+
+Layout_Person := RECORD
+  STRING3  name;
+  UNSIGNED2 age;
+  BOOLEAN good;
+END;
+
+sprayPrepFileName := prefix + 'spray_prep';
+desprayOutFileName := dropzonePath + WORKUNIT + '-spray_input';
+//sprayOutFileName := prefix + 'spray_test';
+
+allPeople := DATASET([ {'foo', 10, 1},
+                       {'bar', 12, 0},
+                       {'baz', 32, 1}]
+            ,Layout_Person);
```

Comment:
Would be better test if initial set was larger and distributed.
Could do something like:
```
allPeople := DATASET(10000, TRANSFORM(layout_Person, SELF.name := (string)COUNTER; SELF.age := COUNTER%99; SELF.good := IF(COUNTER%2=1, TRUE, FALSE)), DISTRIBUTED);
```

Reply:
Yes, but I should test it with SprayFixed().

Reply:
Changed and updated for SprayFix().